import { catchAsyncError } from "./catchAsyncError";







export const isAuthorized=catchAsyncError